api/proto/common/common.proto

```diff
@@ -3,8 +3,47 @@ package common;
 option go_package = "github.com/Layr-Labs/eigenda/api/grpc/common";
 
 message G1Commitment {
-	// The X coordinate of the KZG commitment. This is the raw byte representation of the field element.
-	bytes x = 1;
-	// The Y coordinate of the KZG commitment. This is the raw byte representation of the field element.
-	bytes y = 2;
+  // The X coordinate of the KZG commitment. This is the raw byte representation of the field element.
+  bytes x = 1;
+  // The Y coordinate of the KZG commitment. This is the raw byte representation of the field element.
+  bytes y = 2;
 }
+
+/////////////////////////////////////////////////////////////////////////////////////////////////
+// Messages for WIP RPCs. These are subject to change, do not consider these to be stable API. //
+/////////////////////////////////////////////////////////////////////////////////////////////////
+
+// A unique identifier for a blob.
+message BlobKey {
+  oneof key {
+    AuthHeaderBlobKey auth_header_blob_key = 1;
+    BatchHeaderBlobKey batch_header_blob_key = 2;
+  }
+}
+
+// Uniquely identifies a blob based on its auth header hash.
+message AuthHeaderBlobKey {
+  bytes auth_header_hash = 1;
+}
+
+// Uniquely identifies a blob based on its batch header hash and blob index.
+message BatchHeaderBlobKey {
+  bytes batch_header_hash = 1;
+  uint32 blob_index = 2;
+}
```

Comment:
Is there any missing information here? Do we need the quorum ID?

Reply:
These are sufficient to identify the blob. We probably want to include the quorum ID in the chunk key.

Reply:
Added quorum ID to chunk key.